controllers/spacebindingcleanup/spacebinding_cleanup_controller_test.go

```diff
@@ -64,12 +79,15 @@ func TestDeleteSpaceBinding(t *testing.T) {
 		sbLaraRedhatAdmin := sbLaraRedhatAdmin.DeepCopy()
 		now := metav1.Now()
 		sbLaraRedhatAdmin.DeletionTimestamp = &now
-		reconciler, request, fakeClient := prepareReconciler(t, sbLaraRedhatAdmin, sbJoeRedhatView, sbLaraIbmEdit, laraMur, joeMur, ibmSpace)
+		fakeClient := test.NewFakeClient(t, sbLaraRedhatAdmin, sbJoeRedhatView, sbLaraIbmEdit, laraMur, joeMur, ibmSpace)
+		reconciler := prepareReconciler(t, fakeClient)
 
 		// when
-		_, err := reconciler.Reconcile(context.TODO(), request)
+		res, err := reconciler.Reconcile(context.TODO(), requestFor(sbLaraRedhatAdmin))
 
 		// then
+		require.False(t, res.Requeue)
+		require.True(t, res.RequeueAfter == 0) // no requeue
```

Comment:
Could you please update also the other places?

Reply:
missed those 🤦‍♂️ . Done https://github.com/codeready-toolchain/host-operator/pull/857/commits/355026b34c21e9c54bcf209a3b61533c8bac5387